cloudbaseinit/plugins/common/userdataplugins/cloudconfigplugins/users.py

```diff
@@ -172,4 +178,25 @@ def process(self, data):
                 LOG.warning("An error occurred during user '%s' creation: '%s"
                             % (user_name, ex))
 
+            if ADMINISTRATORS in self._get_groups(item):
+                admin_public_keys = item.get('ssh_authorized_keys', [])
+                administrators_authorized_keys.extend(admin_public_keys)
+
+        if osutils.group_exists(ADMINISTRATORS):
+            program_data_dir = os.getenv("PROGRAMDATA", "C:\ProgramData")
```

Comment:
Wouldn't it be better to not hardcode the system drive letter here, but get it through a `SystemDrive` environment variable too?